ui/package.json

```diff
@@ -53,10 +53,17 @@
     "codegen": "rm -rf src/tanagra-api && openapi-generator-cli generate -i ../api/src/main/resources/api/service_openapi.yaml -g typescript-fetch -t ./openapi-templates --additional-properties=typescriptThreePlus=true -o src/tanagra-api && sed -i 's/AttributeValue;/AttributeValue | null;/g' src/tanagra-api/models/SearchEntityInstancesResponse.ts"
   },
   "eslintConfig": {
-    "extends": ["react-app", "react-app/jest"]
+    "extends": [
```

Comment:
Is there any standard for me to know when should I separate the items in different lines?

Reply:
The standard is defined by prettier. You can run `npx prettier --write` to automatically reformat everything in the repo. That should have no effect on anything you haven't changed because the repo should always be properly formatted. It may not be enforced correctly on submit yet but it will be eventually.